src/frontend/data/sideEffects/createLiveRegistration/index.ts

```diff
@@ -21,5 +28,5 @@ export const createLiveRegistration = async (
   if (!response.ok) {
     throw { code: 'invalid', ...(await response.json()) };
   }
-  return (await response.json()) as LiveRegistration;
+  return response.json();
```

Comment:
If you no longer await here, you can remove the async on this function

Reply:
I'm awaiting 2 lines before.

Reply:
and line 19

Reply:
true, I'm blind